assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -149,13 +153,25 @@ function useUploader() {
         throw createError('SizeError', file.name, message);
       }
 
+      trackEvent('video_transcoding', 'editor', '', '', {
+        file_size: file.size,
+        file_type: file.type,
+      });
+      const trackTiming = getTimeTracker(
+        'video transcoding',
+        'editor',
+        'Media'
+      );
+
       // Transcoding is enabled, let's give it a try!
       try {
         // TODO: Only transcode & optimize video if needed (criteria TBD).
         const newFile = await transcodeVideo(file);
+        trackTiming();
         additionalData.media_source = 'video-optimization';
         return uploadMedia(newFile, additionalData);
       } catch (err) {
+        trackTiming();
```

Comment:
Let's just write this once in a `finally {}` block at the end.

Reply:
I did originally use a finally [here](https://github.com/google/web-stories-wp/pull/6188/commits/f13f6b91dc954724c9d224d4e7cb2a00f44cbdef). But changed as the linter didn't like it. The issue is there is a return on line 169. I can revert the change and ignore the lint, but I think it is correct.

Reply:
Ah, didn't see the return there. Yes you're right, let's leave it then.